README.md

```diff
@@ -58,6 +58,51 @@ make serve
 
 This will start the local Hugo server on port 1313. Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
 
+### Troubleshooting in MacOS for Openfile Problems
+
+If you find error looks like the following error:
+
+```
+Built in 16332 ms
+Watching for changes in /path/to/website/{archetypes,assets,content,data,i18n,layouts,static,themes}
+Watching for config changes in /path/to/website/config.toml, /path/to/website/themes/docsy/config.toml, /path/to/website/go.mod
+Environment: "development"
+Serving pages from memory
+Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
+Web Server is available at http://localhost:1313/ (bind address 127.0.0.1)
+Press Ctrl+C to stop
+ERROR 2020/08/01 19:09:18 Error: listen tcp 127.0.0.1:1313: socket: too many open files
+make: *** [serve] Error 1
+```
+
+We can try to check openfiles current limit:
```

Comment:
Avoid ["we"](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we):
```suggestion
Try checking the current limit for open files:
```